kinopio-club/kinopio-client
# Remove journal space option/noise

goal = spaces feel more like blank pages in a notebook. encourage you to reduce friction for creating new spaces. (good friction reduction = improves your life and mine, mutually beneficial. ties into business design)

# Space list : 
x remove moonPhase
x Remove tweet space code
x today space badge for all spaces

new user experience = lots of bright ‘today’ markers, tone it down?

# Addspace
x List user templates
x new button -> + / options dropdown

# space details info
x when spacename input field focused, show cal button (on top of pinned button?), which appends journal style date to name 
x .active it if date is already in name

# new space/ currentspace/addspace: new space default cards
x no randome pos jitter
x add date card/img, 
x all spaces get moonphase
x current moonphase = system command custom icon in date card (hobonichi inspired)

# Server
x generate date image seperately
x Add :secret Migration route:
x Find all journal users and create or find a group for them.
x Add all their journals to the group

# User prefs 
x rename ‘new spaces are blank’ to new spaces: ‘hide tutorial cards’
x remove all journal, weather, prompt code

# spacefilter 
x remove journal filter
x remove dialogSpaceFilterByType references

# x Templates
x Remove user templates 

# qa 
x check that new space cmd btn still works
x cmd templates btn still works
x sign up w new acct
x create new space and then sign up
x anon also updates changelog count so signing in doesn’t show update notification


CODE REVIEW

SHIP/server
# after shipping 

watch server logs
run prod migration route
Then delete migration route
db changelog



## Blog post 
Journals are replaced by normal spaces , existing journals will be in a ‘Journals’ group for filtering 
https://forum.kinopio.club/t/merging-regular-and-journal-spaces-functionality/1449/13

# api docs
all journals get moonphase attr
remove mentions of journals and weather
add route : GET /space/date-image , latest img url, used for new spaces
remove user.weatherLocation , weatherUnitIsCelcius, showWeather  


# server later clean up tasks

Remove custom prompts table
remove tweet/twitter space code
remove all other journal, weather, prompt code
remove all loadJournalSpace
remove journal ops
remove any user.weatherLocation , weatherUnitIsCelcius, showWeather 
remove weather services route, env var
await queryInterface.removeIndex('journalPrompts', ['userId', 'name']);
await queryInterface.removeColumn('journalPrompts', 'userId')

update api docs
